prow/plugins/label/label.go

```diff
@@ -228,7 +232,7 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	toRepeat := []string{}
 	if len(sigMatches) > 0 {
 		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
+		if isMember {
```

Comment:
Wont this change prevent the bot from repeating mentions made by non-members and instead make the bot repeat mentions by members? I think toRepeat should be set outside this conditional and the conditional should be moved down a few lines so that comments are only created if `!isMember`.